cluster_tools/copy_volume/copy_volume.py

```diff
@@ -81,7 +82,12 @@ def run_impl(self):
             # if this is a label multi-set, the dtypes needs to be changed
             # to be uint64
             is_label_multiset = ds.attrs.get("isLabelMultiset", False)
-            ds_dtype = "uint64" if is_label_multiset else ds.dtype
```

Comment:
This change is a bit unfortunate. `ds_dtype` is the data type of the input dataset and should not be changed here.
Instead you should update how the `dtype` is determined in line 123 and following (i.e. change it to the unsigned dtype if `int_to_uint` is True).